docs: added logging tutorial to quickstart and plugins (#341)
ApeWorX/ape: docs/userguides/quickstart.md

$ ape run deploy
```

> **Ape Logging:** 

Reviewer: ```suggestion
### Logging
```

> If you want to enable debug logging you should appened

Reviewer: ```suggestion
To enable debug logging, run your command with the `--verbosity` flag using `DEBUG` as the value:
```

> If you want to view more information while running any of the ape commands.

Reviewer: Adjusting the log level does not necessarily cause there to be more information - that only really happens when setting to the `DEBUG` level (e.g. `--verbosity DEBUG`). But, you can also get _less_ information by setting the level to one that is less verbose (e.g. `--verbosity ERROR`). 99.9% of the time, I bet, `--verbosity` will be used to just enable debug logging though.

Author: I reworded the statement to incorporate this comment. Please let me know if I understand it properly.

> `ape command_word -v [LVL]` or  `ape command_word --verbosity [LVL]` 

Reviewer: ```suggestion
ape command_word --verbosity DEBUG
```

> ```bash

Reviewer: We can delete this code block

c.doThat(sender=a)  # Make Web3 transactions
assert c.MyEvent[-1].caller == a  # Search through Web3 events
```

> 

Reviewer: Only need one 
```suggestion
```
